.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Kai will go ahead change the contents to be the original content

Reply:
Done @sumitramanga